packages/google_atlas/lib/src/google_atlas.dart

```diff
@@ -147,3 +186,10 @@ class _GoogleMapsProviderState extends State<GoogleMapsProvider> {
     );
   }
 }
+
+class GoogleAtlasDrawings {
```

Comment:
I think this should be included in its file, maybe a `models` folder... thoughts @felangel ?

Reply:
I agree plus I'm not super happy with the name. I don't think a Marker should be considered a Drawing and I also don't think Circles and Markers are related so it seems weird to couple them in a class like this. Thoughts?

Reply:
I see what are you saying. Grouping them in a class was a necessity (at least I couldn't figure another way) due to the fact that `FutureBuilder` it's tied to only a future and that future completion must provide both markers and circles (maybe in the future other objects)..